bcipy/signal/model/pca_rda_kde/pca_rda_kde.py

```diff
@@ -0,0 +1,113 @@
+import pickle
+from pathlib import Path
+from typing import List
+
+import numpy as np
+from ..base_model import ModelEvaluationReport, SignalModel
+from .classifier import RegularizedDiscriminantAnalysis
+from .cross_validation import cost_cross_validation_auc, cross_validation
```

Comment:
Happy to follow if you have a style preference here. 

Just wanted to briefly explain my thinking about the module structure and why I used relative imports in a few places here.

In `bcipy/signal/model/__init__.py`, we'll expose the small subset of classes and functions that folks should interact with when importing from another module (like `bcipy/tasks/...` or something).

Within that module, the files can be agnostic about their absolute positioning, and if we use relative imports it's easy to move them to subfolders, etc.

Similar reasoning for `bcipy/signal/model/pca_rda_kde` as a module - the only exported items will basically be the `PcaRdaKdeModel`, and the other internal parts of that model, such as `ChannelWisePrincipalComponentAnalysis` should be encapsulated from the end user.

In fact, to keep following that convention, the tests for `PcaRdaKdeModel` would probably belong inside the module folder, since they _are_ inspecting these internal components - but I didn't want to make this change without discussion.

Anyway - happy to adopt whatever layout is consistent with the rest of the project and keeps these modules clean!

Reply:
resolved